### PR TITLE
Add location_id to front end location shapes

### PR DESCRIPTION
**Pull request by philip-cline**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Regression where we wouldn't include `location_id` when editing and saving a polygon.

### CODE REVIEW

**Review by amy-corson-ibigroup — APPROVED** [review]
Gorgeous!

File: lib/editor/components/map/LocationsLayer.js [hunk 1]
```diff
@@ -83,7 +83,8 @@ const LocationsLayer = (props: Props) => {
         geometry_id: key,
         geometry_pt_lat: coord.lat,
         geometry_pt_lon: coord.lng,
-        id: index
+        id: index,
+        location_id: activeEntity.location_id
```
Comment (miles-grant-ibigroup) [inline lib/editor/components/map/LocationsLayer.js]: Wait could you possibly add a comment/explain here why this is needed? Why are we doing this

**Review by miles-grant-ibigroup — APPROVED** [review]
All good pending explanatory comment. I have a feeling it might be a multi-line...

File: lib/editor/components/map/LocationsLayer.js (hunk 1 above)
Comment (philip-cline) [reply lib/editor/components/map/LocationsLayer.js]: The backend requires the location_id in each location_shape. We should really be just doing that work on the backend I suppose

Comment (philip-cline) [reply lib/editor/components/map/LocationsLayer.js]: See https://github.com/ibi-group/datatools-ui/pull/936/commits/aaf328f3c513c59337fdaad41937e336d4843faa and let me know if that clears it up for you :)

**Comment by miles-grant-ibigroup** [comment]
Looks great thanks! I think keeping it in the front end is fine -- better to pass it than to have to fish it out, no?

**Comment by philip-cline** [comment]
I agree. Easier to finagle on FE.

**Pull request merged**